playbooks/init/base_packages.yml

```diff
@@ -41,7 +41,7 @@
         - "{{ 'python3-PyYAML' if ansible_distribution == 'Fedora' else 'PyYAML' }}"
         - libsemanage-python
         - yum-utils
-        - "{{ 'python3-docker' if ansible_distribution == 'Fedora' else 'python-docker' }}"
+        - "{{ 'python3-docker' if ansible_distribution == 'Fedora' else 'python-docker-py' }}"
```

Comment:
This change has been merged in 3.11, would you mind rebasing - or rather creating a new PR w/ `ansible_hostname` -> `inventory_hostname` change?

Reply:
@vrutkovs In case you'd like to use it, I've done that at #10981 (and tested that it fixes deployment with Calico).